usr/lib/common/trace.c

```diff
@@ -131,7 +131,7 @@ void set_trace(struct trace_handle_t t_handle)
 
 void trace_finalize(void)
 {
-    if (trace.fd)
+    if (trace.fd >= 0)
```

Comment:
It is cleaner this way, but since we anyway ignore the error from `close`, we could even skip the `if` statement altogether.

Reply:
Well, valgrind issued a warning when close was called with -1, so thats why I wanted to avoid doing that.